tap_hubspot/__init__.py

```diff
@@ -1422,6 +1426,7 @@ def discover_schemas():
         LOGGER.info('Loading schema for Custom Object - %s', custom_stream["stream"].tap_stream_id)
         result['streams'].append({'stream': custom_stream["stream"].tap_stream_id,
                                   'tap_stream_id': custom_stream["stream"].tap_stream_id,
+                                  "table_name": custom_stream["custom_object_name"],
```

Comment:
Is it necessary to have new key `table_name`? Because it looks like standard ojects won't be having that key.

Reply:
Yes, this `table_name` property value will be reflected on the app and in the extracted records. Whereas `stream` will retain the original object name which we need to make API requests.